### PR TITLE
Move data to server

### PR DESCRIPTION
**Pull request by wtharvey2**
A PR to trial http requests for use in storing session data and ID numbers

### CODE REVIEW

**Comment by wtharvey2** [comment]
@nickvazz, can you take a look at this if you have a chance? I implemented an http request in this branch that calls for an ID number when you hit submit off of the initial home page, but am having issues getting the information to the browser JS. I think I'm either failing to attach the ID number properly on the server-side or failing to call the client-side code correctly, but haven't been able to find the issue after a decent amount of digging and attempting to use axios and fetch.

The client code is here: https://github.com/wtharvey2/Decide/blob/move-data-to-server/decideMain.js#L15-L38
Server code is here: https://github.com/wtharvey2/Decide/blob/move-data-to-server/dataComm.js#L30-L35

File: dataComm.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+})
+
+// get a unique ID number
+router.get('/newID', function (req, res) {
+  // get max id number and add 1
+  maxIDNumber += 1;
+  // return the id number
+  newIDnumber = String(maxIDNumber);
+  res.json({ 'id_num': String(500)});
+})
```
Comment (nickvazz) [inline dataComm.js]: Are you returning only `String(500)` here on purpose? Is it for debug?

File: decideMain.html [hunk 2]
```diff
@@ -4,10 +4,10 @@
     <title>decideMain</title>
     <!-- <link rel="stylesheet" href="mainStyle.css"> -->
     <script type='text/javascript'
-            src='decideMain.js'></script>
+            src='decideMainJS'></script>
   </head>
   <body>
-    <form id="groupForm" action="decideQuestion" method="get">
+    <form id="groupForm" onsubmit="makeNewURL()" method="post">
```
Comment (nickvazz) [inline decideMain.html]: This looks like its a post where other stuff is get requests?

**Comment by nickvazz** [comment]
Looks like it might be a post/get request issue as a first pass. How would I build locally, there's a `dockerfile` now, is it all set for production but what about local debug? (or how are _you_ locally debugging 😛)

File: dataComm.js (hunk 1 above)
Comment (wtharvey2) [reply dataComm.js]: @nickvazz It's there for debug only. Normally would return newIDnumber, which should be 1001 the first time it's requested

File: decideMain.html (hunk 2 above)
Comment (wtharvey2) [reply decideMain.html]: Crud. I had updated this, but hadn't pushed to the remote. I'll push now. It will have a different attempt (using axios I think)

**Comment by wtharvey2** [comment]
@nickvazz, I'm debugging just using a node server locally. I'm deploying it in docker on a [forked branch. 
.](https://github.com/wtharvey2-docker/Decide)

File: dataComm.js (hunk 1 above)
Comment (wtharvey2) [reply dataComm.js]: Returning it for debug only. Normally would be newIDnumber

**Comment by wtharvey2** [comment]
I don't think I'm successfully sending anything in the body of this response based on a few print statements that I put in the server side (and have since removed). https://github.com/wtharvey2/Decide/blob/move-data-to-server/dataComm.js#L5-L14

**Pull request merged**